models/skirun.js

```diff
@@ -0,0 +1,18 @@
+// require mongoose
+// set up shorthand Schema variable to stand in for mongoose.Schema
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+// SkiRun schema
+var SkiRunSchema = new Schema({
+  resort: String,
+  skiTrail: String,
+  runRating: String,
+  skierAbility: String,
+  runCondition: String
+});
+
+// SkiRun model
+var SkiRun = mongoose.model('SkiRun', SkiRunSchema);
+
+module.exports = SkiRun;
```

Comment:
This little red thing at the end of the line means you don't have a blank line at the end of your file... it's customary to have a blank line at the end of your files.